src/controls.css

```
@@ -0,0 +1,40 @@
+.controls-title {
+  font-size: 40px;
+  margin-bottom: 20px;
+}
+
+.controls i{
+  color: cadetblue;
+  margin: 0 0.6rem;
+}
+
+.controls i:hover{
+  color: cornflowerblue;
+}
+
+.controls ol{
+  padding-left: 10px;
+}
+
+.controls li{
+  display: flex;
+  align-items: center;
+  font-size: 20px;
+}
+
+p{
```

```suggestion
p {
```

Repeat the same for all the CSS rules which have no space between selector and `{`